### PR TITLE
Update with question about Brexit

### PR DESCRIPTION
**Pull request by NickCEBM**
FAQ update with a section about future adjustments we will do for Brexit.

### CODE REVIEW

File: euctr/templates/about.html [hunk 1]
```diff
@@ -232,6 +232,20 @@ <h2 id="part-of">What Happens when Sponsors Merge or Split?</h2>
...
+        <h2 id="part-of">What is the impact of Brexit?</h2>
+            
+        <p>Due to Brexit, as of 1 January 2021, new trials from the UK will no longer be registered on the EUCTR.
+        Any UK protocols that were in "Ongoing" status as of the end of 2020 are now locked in the status "GB - no longer in EU/EEA"
+        and can no longer be updated. This means ongoing trials registered exclusively in the UK will never
+        have completion data appear. While sponsors may still upload results, we will not be able to reliably
+        assess the due date of these trials. For trials registered in more than one location that include the UK,
+        we will ignore the UK protocols in status "GB - no longer in EU/EEA" when determining if a trial is complete
+        or not. As of January 2021, this has not yet been implemented but we are working to adjust our code to
+        incoroprate these changes as soon as possible. When this is implemented, some "Not Due - Ongoing" trials will move
```
Comment (madwort) [inline euctr/templates/about.html:245]: typo `incorporate`

File: euctr/templates/about.html [hunk 2]
```diff
@@ -232,6 +232,20 @@ <h2 id="part-of">What Happens when Sponsors Merge or Split?</h2>
         acquisitions; if you think we should update any of our data on
         this, please <a href="#contact">tell us</a>.
         </p>
+        
+        <h2 id="part-of">What is the impact of Brexit?</h2>
```
Comment (madwort) [inline euctr/templates/about.html:236]: the `id` in the `<h2>` is to allow others to link directly to this section of the page & should therefore be unique on the page. e.g. https://eu.trialstracker.net/about#major-sponsor , https://eu.trialstracker.net/about#inconsistent-data

Anyway, this unique id `part-of` already occurs three times! We can fix the others another time, but how about we do the right thing in this PR & make this something like:

`<h2 id="brexit">What is the impact of Brexit?</h2>`

**Review by madwort — CHANGES_REQUESTED** [review]
minor corrections then good to go

File: euctr/templates/about.html (hunk 2 above)
Comment (NickCEBM) [reply euctr/templates/about.html]: Ohhh, I'm a complete HTML newbie as far as what the functionality of anything is, so this is just undoubtedly just me copy/pasting blinding without regard for this! I've gone through and adjusted throughout with this in mind!

File: euctr/templates/about.html (hunk 1 above)
Comment (NickCEBM) [reply euctr/templates/about.html]: Thanks!

**Review by madwort — APPROVED** [review]
looks good to me

File: euctr/templates/about.html (hunk 2 above)
Comment (madwort) [reply euctr/templates/about.html]: ah cool, thanks for fixing!

**Pull request merged**